apps/pie-storybook/stories/pie-button.stories.ts

```diff
@@ -317,8 +317,9 @@ const FormTemplate: TemplateFunction<ButtonProps> = (props: ButtonProps) => html
     </form>
     <script>
         // Display a success message to the user when they submit the form
-        const form = document.querySelector('#testForm');
-        const formLog = document.querySelector('#formLog');
+        var form = document.querySelector('#testForm');
```

Comment:
There's currently an issue where navigating from one form story to another (or indeed back to the same story) within the same session causes an error where the second form isn't registered because a variable called `form` (or `formLog`) has already been assigned, and storybook doesn't properly dispose of these.

Using `var` allows the variable to be reassigned, preventing the issue.